satpy/tests/test_scene.py

```diff
@@ -21,6 +21,8 @@
 import unittest
 from unittest import mock
 
+from satpy.tests.utils import make_dsid, make_cid, default_id_keys_config, make_dsq
```

Comment:
Here and elsewhere — usually we import within the test function (I think we do this so that an import failure will be a test failure, not a test collection failure), why not now?

Reply:
These are utility functions for tests, so nothing we need to mock or expect import failure on.